LLVMFloatToFixed/LLVMFloatToFixedPass.cpp

```diff
@@ -55,6 +55,8 @@ bool FloatToFixed::runOnModule(Module &m)
   closePhiLoops();
   cleanup(vals);
 
+  convertIndirectCalls(m);
```

Comment:
It would be nice if this was called from inside performConversion() but I suspect there are reasons not to do that.

Reply:
The main reason is that we expect to perform the indirectCallConversion after the base conversion, together with the related cleanups, is already done. 

In case we try to perform the conversion before the cleanup, we have various errors such as compilation error during cleanup (an `eraseFromParent()` explodes, or `an Assertion `!NodePtr->isKnownSentinel()' failed.`).

Having it outside performConversion() and after the cleanups allow for an easier handling of the trampoline functions altogether. 

Moreover, the main idea of this would be to "conceptually" separate the OpenMP integration from the conversion pass - in particular it would be quite easy right now to use separate passes both before Init and after Conv, and although not super necessary in our opinion right now (since the integration is quite small), it may be helpful to do in the future.